panoptes_client/panoptes.py

```diff
@@ -400,6 +399,25 @@ def get_bearer_token(self):
             )
         return self.bearer_token
 
+    def bearer_expired(self):
```

Comment:
could this be renamed to `def valid_bearer_token(self)` and maybe extract out the `if self.bearer_token is None` check to its own method (probably overkill but might be useful in other places).